### ISSUE TITLE
No name 'Cluster' in module 'cassandra.cluster't(no-name-in-module)

### ISSUE THREAD

**Issue by raph90** [PyCQA/pylint#4082]
### Steps to reproduce
1. VSCode using Pylance
2. Conda environment, VSCode is set to the correct interpreter, with cassandra-driver installed and pylint

### Current behavior
cassandra.cluster.Cluster is being imported, but Pylint says it doesn't exist

### Expected behavior
Pylint should say that it exists. There are no problems with actually importing it, or importing any other cassandra modules.
No errors are shown on the other modules.

EDIT

The issue was to do with how VSCode was using pylint. I solved the issue with this adding this to my pylintrc:
[MASTER]
init-hook="from pylint.config import find_pylintrc;
import os, sys; sys.path.append(os.path.dirname(find_pylintrc()))"

**Comment by raph90** [comment]
This was not actually fixed-  there was an error in my init-hook which was just disabling Pylint, rather than making it find the package. 

To reiterate, the package exists on my pythonpath, and so far as I can tell there is no other package interfering. I am using the correct interpreter (using miniconda) in VS Code. It's not a language server problem either because I've tried both Jedi and Pylance and both are showing the error.

**Comment by raph90** [comment]
Closing this again - if anyone has this problem, the issue I'm pretty sure is that cassandra-driver includes some C files which pylint doesn't like. You need to add it to extension-pkg-whitelist. 

extension-pkg-whitelist=cassandra

**Comment by hippo91** [comment]
@raph90 thanks for you report and comments.

**Comment by ashb** [comment]
Further to this (possibly a separate bug):


This fails:

```
pylint  --rcfile=/dev/null --extension-pkg-whitelist=cassandra  airflow/providers/apache/cassandra/hooks/cassandra.py
```

But this passes

```
pylint  --rcfile=/dev/null --extension-pkg-whitelist=cassandra  airflow.providers.apache.cassandra.hooks.cassandra
```
